pkgs/development/libraries/xine-lib/default.nix

```diff
@@ -33,7 +35,21 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-71GyHRDdoQRfp9cRvZFxz9rwpaKHQjO88W/98o7AcAU=";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
Will these patches be incorporated in the future?

Reply:
I just checked upstream Mercurial repo and they do have those changes in their tree.

libcaca fix is https://sourceforge.net/p/xine/xine-lib-1.2/ci/3f383e140270cd45448bd9ebaa08232579e1c1af/

ffmpeg fix is https://sourceforge.net/p/xine/xine-lib-1.2/ci/4e5ad7ebed6111ba9e1ea2eab4a6b94477b066a8/

Reply:
As I have seen here, sourceforge has no native way to [download patch files](https://sourceforge.net/p/forge/feature-requests/614/).

Just to make sure, and [to make hammer happy](https://github.com/jtojnar/nixpkgs-hammering), comment why these patches are needed, plus an advice to remove them at the next release.

Reply:
Done